wherehows-dao/src/main/java/wherehows/dao/view/BaseViewDao.java

```diff
@@ -22,54 +22,24 @@
 import lombok.SneakyThrows;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import wherehows.models.view.DatasetOwner;
-import wherehows.models.view.DatasetColumn;
-
-import static org.apache.commons.lang3.StringUtils.isNotBlank;
 
 
 /**
- * Intended to be used for Front-end readonly functions
+ * Base DAO class for view (read only) model, containing generic functions
  * Use native SQL and then map to entity
  */
-public class MetadataReadOnlyDao {
+public class BaseViewDao {
 
-  private static final Logger log = LoggerFactory.getLogger(MetadataReadOnlyDao.class);
+  private static final Logger log = LoggerFactory.getLogger(BaseViewDao.class);
 
   private final EntityManagerFactory _emFactory;
 
-  public MetadataReadOnlyDao(EntityManagerFactory factory) {
+  public BaseViewDao(EntityManagerFactory factory) {
     this._emFactory = factory;
   }
 
   private static final String GET_DATASET_URN_BY_ID = "SELECT urn FROM dict_dataset WHERE id = :id";
```

Comment:
Not sure if we want to keep this is BaseView?

Reply:
Removed that from BaseViewDao